.circleci/config.yml

```diff
@@ -16,7 +16,7 @@ cpu: &cpu
   environment:
     TERM: xterm
   machine:
-    image: default
+    image: ubuntu-1604:201903-01
```

Comment:
question: why are we changing the image to ubuntu? why not use the default container